Mathlib/Algebra/Module/Submodule/Basic.lean

```diff
@@ -344,14 +344,11 @@ theorem coe_mem (x : p) : (x : M) ∈ p :=
 variable (p)
 
 instance addCommMonoid : AddCommMonoid p :=
-  { p.toAddSubmonoid.toAddCommMonoid with
-    add := (· + ·)
-    zero := 0 }
+  { p.toAddSubmonoid.toAddCommMonoid with }
```

Comment:
Why not just `:= p.toAddSubmonoid.toAddCommMonoid` here? Or `by _infer_instance`? Probably we should just delete this instance.

Reply:
Trying this out

Reply:
These stressed the elaborator in a couple of places but otherwise seem fine to me.